index.css

```diff
@@ -788,3 +789,133 @@ h3 {
   border: 1px solid #f55800;
   box-sizing: border-box;
 }
+
+.container_popUp {
+  position: fixed;
+  top: 60px;
+  display: flex;
+  flex-direction: column;
+  width: calc(100vw - 32px);
+  margin-left: 4px;
+  z-index: 10;
+  background-color: #fff;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  height: calc(100vh - 60px);
+}
```

Comment:
- [ ] Please make sure that your popup is visible in the mobile version. Right now, there's no button to pop the modal, so work on this issue.

Reply:
Addressed 🔥